velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -276,6 +276,10 @@ void PageReader::prepareDataPageV1(const PageHeader& pageHeader, int64_t row) {
           pageData_,
           pageData_ + defineLength,
           arrow::bit_util::NumRequiredBits(maxDefine_));
+      wideDefineDecoder_ = std::make_unique<arrow::util::RleDecoder>(
+          reinterpret_cast<const uint8_t*>(pageData_),
+          defineLength,
+          arrow::bit_util::NumRequiredBits(maxDefine_));
```

Comment:
Create wideDefineDecoder_ in page reader in case of segment fault at [PageReader.cpp#L546-L547](https://github.com/facebookincubator/velox/blob/main/velox/dwio/parquet/reader/PageReader.cpp#L546-L547).

Reply:
Makes sense. Fixed.